bitnami/magento/Chart.yaml

```
@@ -8,11 +8,11 @@ dependencies:
     repository: https://charts.bitnami.com/bitnami
     tags:
       - magento-database
-    version: 9.x.x
+    version: 10.x.x
```

Please note there is a new major in the database, see https://github.com/bitnami/charts/pull/9827
```suggestion
    version: 11.x.x
```